client/src/components/main/PostList.jsx

```diff
@@ -4,14 +4,24 @@ import { connect } from 'react-redux';
 import { Grid, Row, Col, Panel } from 'react-bootstrap';
 import PostListEntry from './PostListEntry.jsx';
 
-const PostList = ({posts}) => {
-  return (
-    <div>
-      {posts.map((post, i) => { 
-        return <PostListEntry post={post} id={i} key={i} />; 
-      })}
-    </div>
-  );
+const PostList = ({posts, type}) => {
+  if (type === 'TYPE_PROFILE') {
+    return (
+      <div>
+        {posts.map((post, i) => { 
+          return <PostListEntry post={post} id={i} key={i} />; 
+        }).slice(0, 3)}
```

Comment:
Why is this being sliced?

Reply:
This is a naive approach to displaying only the first three posts.